apps/src/lib/script-editor/LevelToken.jsx

```diff
@@ -0,0 +1,161 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {Motion, spring} from 'react-motion';
```

Comment:
Knowing what we know now, I prob wouldn't have gone with `react-motion` originally: https://spectrum.chat/react/general/the-state-of-animation-in-react-introducing-react-spring~6ddc2207-0953-4c93-934f-3916414d287d